File: OCH_Tao/ABC/A/028.js
const input = parseInt((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0], 10);
if (input === 100) {
  console.log("Perfect");
} else if (input >= 90) {
  console.log("Great");
} else if (input >= 60) {
  console.log("Good");
} else {
  console.log("Bad");
}

File: OCH_Tao/ABC/A/028.cpp
#include <bits/stdc++.h>
using namespace std;
int main()
{
  int N;
  cin >> N;
  if (N == 100)
  {
    cout << "Perfect" << endl;
  }
  else if (N >= 90)
  {
    cout << "Great" << endl;
  }
  else if (N >= 60)
  {
    cout << "Good" << endl;
  }
  else
  {
    cout << "Bad" << endl;
  }
}